src/keboola/component/dao.py

```diff
@@ -645,7 +645,7 @@ def name(self) -> str:
 
     def __get_stage_inferred(self):
```

Comment:
Tohle asi muzeme zahodit a udelat to jako abstract metodu. Mam taky pocit ze self.__stage se nikde uz nepouziva, je to nejaky pohrobek. TableDefinition to ted pretezuje a pouziva existenci uri. A NEBO Mozna by to slo zobecnit na existenci `id` ktery je jak u Table tak u File def. Dokud nezacne connection vracet manifest type u kazdyho input manifestu